bookmarks.html

```diff
@@ -1,41 +1,81 @@
-    <link rel="stylesheet" href="styles.css">
+<html lang="en" data-js="htmlGoesDark">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
     <title>Quizzen-mit-Wizzen</title>
-   
-  
-</head>
+    <script src="./index.js" defer></script>
+  </head>
 
-<body>
+  <body class="body" data-js="bodyGoesDark">
+    <h1 class="header">Quizzen mit Wizzen</h1>
 
-    <h1>Quizzen mit Wizzen</h1>
+    <h2 class="Musik-h2">Musikfragen</h2>
 
-   <h2 class="Musik-h2">Musikfragen</h2>
+    <h2 class="Geschichte-h2">Geschichtsfragen</h2>
 
-   
+    <h2 class="Geografie-h2">Geografiefragen</h2>
 
-   <h2 class="Geschichte-h2">Geschichtsfragen</h2>
+    <h2 class="Klatsch-h2">Klatschfragen</h2>
 
+    <h2 class="Zufall-h2">Zufallsfragen</h2>
 
-<h2 class="Geografie-h2">Geografiefragen</h2>
+    <h2 class="Bio-h2">Biologiefragen</h2>
+  </body>
 
-<h2 class="Klatsch-h2">Klatschfragen</h2>
-
-<h2 class="Zufall-h2">Zufallsfragen</h2>
-
-<h2 class="Bio-h2">Biologiefragen</h2>
-
-</body>
-
-<footer class="nav" aria-labelledby="navigation"> 
-    <a href="index.html" ><button class="Home" aria-label="Homebutton"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" height="3em" width="3em"><path d="M19.07 4.93C17.22 3 14.66 1.96 12 2c-2.66-.04-5.21 1-7.06 2.93C3 6.78 1.96 9.34 2 12c-.04 2.66 1 5.21 2.93 7.06C6.78 21 9.34 22.04 12 22c2.66.04 5.21-1 7.06-2.93C21 17.22 22.04 14.66 22 12c.04-2.66-1-5.22-2.93-7.07M17 12v6h-3.5v-5h-3v5H7v-6H5l7-7 7.5 7H17Z"/></svg></button></a>
-    
-    <a href="bookmarks.html" ><button class="Favoriten" aria-label="Favoritenbutton"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" height="3em" width="3em"><path d="M5 3h14a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2m7 14 .72-.66C15.3 14 17 12.46 17 10.57c0-1.54-1.21-2.75-2.75-2.75-.87 0-1.7.41-2.25 1.05a3.007 3.007 0 0 0-2.25-1.05C8.21 7.82 7 9.03 7 10.57c0 1.89 1.7 3.43 4.28 5.77L12 17Z"/></svg></button></a>
-    
-    <a href="#"><button class="Profil" aria-label="Profilbutton"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" height="3em" width="3em"><path d="M11.5,14C15.64,14 19,15.57 19,17.5V20H4V17.5C4,15.57 7.36,14 11.5,14M18,17.5C18,16.12 15.09,15 11.5,15C7.91,15 5,16.12 5,17.5V19H18V17.5M11.5,5C13.43,5 15,6.57 15,8.5C15,10.43 13.43,12 11.5,12C9.57,12 8,10.43 8,8.5C8,6.57 9.57,5 11.5,5M11.5,6C10.12,6 9,7.12 9,8.5C9,9.88 10.12,11 11.5,11C12.88,11 14,9.88 14,8.5C14,7.12 12.88,6 11.5,6Z" /></svg></button></a>
- </footer>
- </html>
\ No newline at end of file
+  <footer class="nav" aria-labelledby="navigation">
+    <ul class="footer-ul">
+      <li class="footer-ul__symbol">
+        <a href="index.html" class="footer-ul__home-link"
+          ><button class="Home" aria-label="Homebutton">
```

Comment:
class Names should be camelCase (start with small letter)

`button class="home" aria-label="homebutton"`